src/app/report/edit-display/edit-display.component.ts

```diff
@@ -80,12 +81,11 @@ export class EditDisplayComponent {
   rerunReport(): void {
     let reportId: string = this.report.storageId;
     this.httpService.runDisplayReport(reportId, this.currentView.storageName).subscribe((response) => {
-      let element = document.querySelector('#showRerunResult')!;
       if (this.report == response) {
-        element.setAttribute('style', 'background-color: green');
+        this.rerunBackgroundColor = 'background-color: green';
         this.rerunResult = '[Rerun succeeded]';
       } else {
-        element.setAttribute('style', 'background-color: red');
+        this.rerunBackgroundColor = 'background-color: red';
         this.rerunResult = '[Rerun failed]';
       }
```

Comment:
When only using the boolean attribute you can refactor this method to only set the boolean
```ts
this.rerunSuccess = this.report == response
```

Reply:
changed it to the suggested line